tests/contract_testcases/substrate/functions/virtual_function_call_01.sol

```diff
@@ -0,0 +1,9 @@
+abstract contract A {
+    constructor() {
+        _virtual();
+    }
+
+    function _virtual() internal virtual;
+}
+
```

Comment:
This test should ensure that a concrete contract with A as a base actually works, i.e. adding:

```solidity
contract B is A {
	function _virtual() internal override {}

	function m() public {}
}
```
(It does, I've just tested it)

Reply:
Good idea,  thanks